### ISSUE TITLE
Current version doesn't support dialogs

### ISSUE THREAD

**Issue by pr0t3us** [jwisniewski/android-developer-assistant#3]
Thank you for the great app!

Having one issue: At the moment unable to inspect views from dialogs. The Layout Inspector, however, is seeing them fine under `com.android.internal.policy.PhoneWindow$DecorView` root.

**Comment by jwisniewski** [comment]
Thank you and apologies for the issue you have experienced.

I have re-tested the app (v1.1.1) with a few dialogs both from third party apps and the dialogs being a part of the Android operating system. The app was able to inspect them correctly.

I believe the issue must be related to the specific Android version or manufacturer or a kind of apps. Could you share more details? An example dialog (maybe from a concrete app) which I could try to test on my side?

If you wouldn't like to share this information publicly, you can consider sending an email to support (at) appsisle.com

**Comment by pr0t3us** [comment]
Yes, you're right, I was able to find it in the `Element` and `Hierarchy` tab, however, it's hard to select the dialog if the screen has multiple elements in the background(even with the visible filter enabled because the background is semi obscured). It would be great if we can filter out the elements by the viewgroup as the option in the `Hierarchy` tab.

**Comment by jwisniewski** [comment]
Thanks for the suggestion, I will think on that. At the moment, just try to tap multiple times exactly in the same place, the app will iterate, selecting subsequent views which occupy space tapped with a finger.

I hope it helps a bit?

**Comment by Intellectual777** [comment]
#
[Remote_sensing_of_lake_and_river_ice.pdf](https://github.com/jwisniewski/android-developer-assistant/files/12310749/Remote_sensing_of_lake_and_river_ice.pdf)

**Comment by Intellectual777** [comment]
/

>
